source/includes/usage-examples/write-code-examples.java

```diff
@@ -0,0 +1,87 @@
+# start-insert-one
+Document document = new Document("<field name>", "<value>");
+Mono<InsertOneResult> result = Mono.from(collection.insertOne(document));
```

Comment:
Let's document somewhere on the page that all operations that return a `Publisher` don't actually happen unless the returned `Publisher` is subscribed to. Calling `Mono.block` as suggested in a another comment is a way to subscribe.

Reply:
✅ Done.